schema/v5.0/CVE_JSON_5.0.schema

```diff
@@ -720,25 +732,26 @@
                             ],
                             "properties": {
                                 "lang": {
-                                    "type": "string",
-                                    "description": "ISO 639-2 language code",
-                                    "minLength": 1
+                                    "$ref": "#/definitions/language"
                                 },
                                 "description": {
                                     "type": "string",
                                     "description": "string description of problemType, or title from CWE, OWASP",
-                                    "minLength": 1
+                                    "minLength": 1,
+                                    "maxLength": 4000
                                 },
                                 "cweId": {
                                     "type": "string",
                                     "description": "CWE ID of the CWE that best describes this problemType entry",
                                     "minLength": 5,
+                                    "maxLength": 9,
                                     "pattern": "^CWE-[0-9]+$"
```

Comment:
AI:Chris. What is the official regex for CWE?

Reply:
CWE Team confirmed that the numeric range of CWE-99999 is more than reasonable for timelines this needs to support. 
Additionally, had confirmation that the regex should NOT allow a leading 0.

Reply:
updated regex to prevent leading 0 in ID string